viz_scripts/ebike_specific_metrics.ipynb

```diff
@@ -37,11 +39,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pandas as pd\n",
-    "import numpy as np\n",
+    "from collections import defaultdict\n",
+    "import datetime\n",
     "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
     "import seaborn as sns\n",
-    "from collections import defaultdict\n",
+    "\n",
+    "import emission.core.get_database as edb\n",
+    "import emission.core.wrapper.entry as ecwe\n",
+    "import emission.storage.decorations.analysis_timeseries_queries as esda\n",
+    "import emission.storage.decorations.trip_queries as esdt\n",
+    "import emission.storage.decorations.timeline as esdl\n",
+    "import emission.storage.timeseries.abstract_timeseries as esta\n",
+    "import emission.storage.timeseries.timequery as estt\n",
+    "from plots import *\n",
```

Comment:
Can you clarify where the code that uses these is added? I thought this was all already handled in `scaffolding` and `plots`? Why do we need to re-import all of these when they are not actually used?

Reply:
These are unnecessary, removed them now.